fellow-submissions/devonte-duncan/index.html

```diff
@@ -0,0 +1,105 @@
+
+<!DOCTYPE html>
+<html lang=en>
+<head>
+<title>Devy's Quiz</title>
+<meta charset=UTF-8>
+<link rel="stylesheet" href="styles.css" type="text/css" />
+</head>
+<body>
+  <form action="#" method="post">
+     <!--Personal INFO-->
+    <fieldset>  
+      <legend>Personal Info: </legend>
+       <section>
+        <div class="personal-info">
+            <label for="name"><p>Name:</p></label>
+            <input placeholder="Full Name" type="text" id="name" name="user_name">
+        </div>
+        <div class="personal-info">
+            <label for="mail"><p>E-mail</p><label>
+            <input placeholder="Gmails Only" type="email" id="mail" name="user_mail">
+        </div>
+        <div class="personal-info">
+          <p>Birthday:</p>
+              <input type="date" name="bday" max="2001-01-02">
+        </div>
+       </section>
+    </fieldset>
+    
+    <fieldset>
+      <legend>The Important Stuff: </legend>
+      
+       <div class="options">
+        <p class="question">Who would win in a fight to the death?</p>
+        <select name="Fight">
+          <option value="placeholder">Choose your fighter</option>
+          <option value="saitama">Saitama</option>
+          <option value="goku">Goku</option>
+          <option value="vegeta">Vegeta</option>
+          <option value="naruto">Naruto</option>
+        </select>
+       </div>
+       
+       <div class="options">
+           <p class="question">Pick one to eat for the rest of your life.</p>
+          <aside>
+              <input type="radio" name="food" value="pizza"> Pizza<br>
```

Comment:
Get rid of these `<br>` and use `padding` and `margin` as needed. And `Pizza` and `Pasta` should be a `<label>`